packages/sanity/src/core/studio/components/navbar/presence/PresenceMenuItem.tsx

```diff
@@ -79,20 +78,12 @@ export const PresenceMenuItem = memo(function PresenceMenuItem(props: PresenceLi
     >
       <Flex align="center">
         <AvatarCard>
-          <UserAvatar size={1} key={presence.user.id} user={presence.user} />
+          <UserAvatar size={1} key={presence.user.id} user={presence.user} blur={!hasLink} />
```

Comment:
Instead of adding a new `blur` prop to `UserAvatar` – I suggest using the already existing `status` property.

```ts
<UserAvatar status={hasLink ? 'editing' : 'inactive'} />
```

Reply:
Uuh thank you!!